voice/retrieve-info-for-all-calls.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+import nexmo
+from pprint import pprint
+from datetime import datetime, timedelta
+
+client = nexmo.Client(
+    application_id=APPLICATION_ID,
+    private_key=APPLICATION_PRIVATE_KEY_PATH,
+)
+
+NOW = datetime.utcnow()
+DATE_END = NOW.replace(microsecond=0).isoformat()+"Z"
+DATE_START = (NOW - timedelta(hours=24, minutes=00)).replace(microsecond=0).isoformat()+"Z"
+
+params = {"date_start": DATE_START, "date_end": DATE_END}
+response = client.get_calls(params)
```

Comment:
These two lines could just be:
```python
response = client.get_calls(date_start=DATE_START, date_end=DATE_END)
```
... which would be more consistent with the calling conventions above - but ultimately I'm not bothered too much.